src/Hidden/HiddenCss.js

```diff
@@ -89,13 +88,7 @@ function HiddenCss(props: AllProps) {
     className.push(classes[`only${capitalizeFirstLetter(only)}`]);
   }
 
-  if (!React.isValidElement(children)) {
-    return null;
-  }
-
-  return React.cloneElement(children, {
-    className: classNames(children.props.className, className.join(' ')),
-  });
+  return <span className={className}>{children}</span>;
```

Comment:
What do you think of adding a warning if `...other` is non empty?

Reply:
I'll alter the current warning, since a ref is no longer valid there.